### DISCUSSION TITLE
Cross Site Scripting (DOM) issue

### DISCUSSION THREAD

**Discussion by bhansa** [jquery/jquery#5369] [Q&A] [answered]
### Description ###

We recently had code security scans at my firm using a third party tool (Fortify). We found Critical "Cross Site Scripting (Dom) error.
After some digging I found that the error was coming from Jquery which was embedded.

Jquery version:
jQuery v3.6.1

### Ask ###

I want to understand if this is a valid scenario, and does jquery have such vulnerability. If yes, which functions to be specific.

Your suggestions and discussion will be valuable.

**Comment by mgol** [comment, marked as answer]
Moving this to Discussions.

jQuery 3.6.1 has no known security vulnerabilities. Errors you’re getting likely come from how your code is using jQuery. We can’t debug your app for you, though, and we certainly cannot do anything with such few details.

**Reply by bhansa** [reply]
Thank you for prompt response and confirmation!
